index.js

```diff
@@ -0,0 +1,60 @@
+const popUp = document.querySelector('.popUp');
+const hamburger = document.querySelector('.hamburger');
+const popUpCancel = document.querySelector('.popUpCancel');
+
+hamburger.addEventListener('click', () => {
+  popUp.classList.add('active');
+  popUp.style.display = 'flex';
+});
+
+function closePopUp() {
+  popUp.classList.add('hidden');
+  popUp.style.display = 'none';
+}
+
+popUpCancel.addEventListener('click', closePopUp);
+
+const popUpItem = document.querySelector('.popUpItems');
+
+popUpItem.addEventListener('click', closePopUp);
+
+// speakers----
+
+const speakerArea = document.querySelector('.speakerArea');
+
+const speakers = [
+  {
+    speakersImages: 'BE.jpg',
+    speakersName: 'Bülent Ersoy',
+    speakersTitle: 'Turkish Diva',
+    speakersTopic: 'How similar fantasy literature and fantasy music',
+  },
+  {
+    speakersImages: 'Chuk.jpeg',
+    speakersName: 'Chuck Palahniuk',
+    speakersTitle: 'Writer the Fight Club',
+    speakersTopic: 'A different look into Middle-earth',
+  },
+];
```

Comment:
- [ ] Kindly add more speakers to have a minimum of 6 speakers as it is in the Behance design.

![image](https://user-images.githubusercontent.com/86318284/212126952-a0227c9b-9314-412c-97a5-7d857ed5ffbe.png)